docs/modules/operation/pages/deep-dive/notifications/concepts.adoc

```diff
@@ -1,114 +1,94 @@
 
-
 [[ga-notifications-concepts]]
 = Concepts
-#restructure and move some to intro., cross-references to user chapter#
-
-Notifications are how {page-component-title} informs users about an event that happened in the network, without the users having to log in and look at the UI.
-The core concepts required to understand notifications are:
 
-* Events and UEIs
-* Users, Groups, and On-Call Roles
-* Duty Schedules
-* Destination Paths
-* Notification Commands
+This page describes core concepts related to {page-component-title} notifications.
+They ultimately fit together to form an xref:operation:deep-dive/events/event-definition.adoc[Event Notification Definition].
 
-These concepts fit together to form an Event Notification Definition.
-Also related, but presently only loosely coupled to notifications, are Alarms and Acknowledgments.
+NOTE: Alarms and Acknowledgements are also related, but are currently only loosely coupled with notifications.
 
```

Comment:
Only remove if the previous suggestion is accepted. (wouldn't let me include as a multi-line suggestion)
```suggestion
```